ui/component/or-asset-tree/src/or-add-asset-dialog.ts

```diff
@@ -211,32 +256,44 @@ export class OrAddAssetDialog extends LitElement {
             optionalAttributes: AttributeDescriptor[] | undefined = assetTypeInfo?.attributeDescriptors?.filter(e => !!e.optional);
 
         return html`
-            <or-icon style="--or-icon-fill: ${descriptor.colour ? "#" + descriptor.colour : "unset"}" id="type-icon" .icon="${descriptor.icon}"></or-icon>
-            <or-translate style="text-transform: capitalize; margin-bottom: 1.5em" id="type-description" .value="${Util.getAssetTypeLabel(descriptor)}"></or-translate>
+            <div id="type-title">
+                <or-icon style="--or-icon-fill: ${descriptor.colour ? "#" + descriptor.colour : "unset"}" id="type-icon" .icon="${descriptor.icon}"></or-icon>
+                <or-translate id="type-description" .value="${Util.getAssetTypeLabel(descriptor)}"></or-translate>
+            </div>
+            <div id="name-wrapper">
+                <or-mwc-input id="name-input" .type="${InputType.TEXT}" min="1" max="1023" required .label="${i18next.t("name")}" .value="${this.name}" @or-mwc-input-changed="${(e: OrInputChangedEvent) => this.onNameChanged(e.detail.value)}"></or-mwc-input>
+                <div id="parent-wrapper">
+                    <or-mwc-input id="parent" .type="${InputType.TEXT}" readonly .label="${i18next.t("parent")}" .value="${parentStr}" @click="${() => this._onToggleParentAssetSelector()}"></or-mwc-input>
+                    <or-mwc-input id="remove-parent" ?disabled="${!this.parent}" type="${InputType.BUTTON}" icon="close" @click="${() => this._onDeselectClicked()}"></or-mwc-input>
+                    <or-mwc-input id="toggle-parent-selector" icon="${this.showParentAssetSelector ? "pencil-off" : "pencil"}" type="${InputType.BUTTON}" @click="${() => this._onToggleParentAssetSelector()}"></or-mwc-input>
+                </div>
+            </div>
             
             ${!attributes
                 ? html``
                 : html`
-                    <div style="margin-top: 0.5em">
-                        <strong>${i18next.t("attribute_plural")}</strong>
+                    <div>
+                        <div class="heading">${i18next.t("attribute_plural")}</div>
```

Comment:
Actually while at it, I think it would be better to do h1/h2/h3/h4/etc, but I don't think they're well defined yet.